lib/web_ui/lib/src/engine/text_editing.dart

```diff
@@ -305,6 +308,18 @@ class TextEditingElement {
     _subscriptions
       ..add(html.document.onSelectionChange.listen(_handleChange))
       ..add(domElement.onInput.listen(_handleChange));
+
+    // In Firefox, when cursor moves, nor selectionChange neither onInput
+    // events are triggered. We are listening to keyup event to decide
+    // if the user shifted the cursor.
+    // See [_SelectionChangeDetection].
```

Comment:
typo: the class name doesn't contain an underscore.

Reply:
Done.